install/kubernetes/helm/subcharts/mixer/templates/config.yaml

```diff
@@ -714,7 +764,44 @@ spec:
       - destination_service_name
       - destination_service_namespace
       - connection_security_policy
-      - response_flags
```

Comment:
this seems like a potential issue. I think that this *should not* be deleted.

Reply:
Thanks. Added it back.